components/playlist/browser/resources/components/header.tsx

```diff
@@ -152,18 +238,59 @@ export default function Header ({ playlistId }: HeaderProps) {
             nameColor={color.text.primary}
             detailColor={color.text.secondary}
           />
-          <ContextualMenuAnchorButton items={contextualMenuItems} />
+          {contextualMenuItems.length ? (
+            <ContextualMenuAnchorButton items={contextualMenuItems} />
+          ) : (
+            // We need to this so that 0 is not rendered.
```

Comment:
Optional: You can use the `!!` boolean conversion instead of a ternary, if you like:
```suggestion
           {!!contextualMenuItems.length && <ContextualMenuAnchorButton items={contextualMenuItems} />}
```

Reply:
Awesome! That's exactly what I wanted to do 👍